src/pygats/pygats.py

```diff
@@ -725,10 +725,11 @@ def run(funcs: List[str], counter: Optional[int] = 1, output: Optional[str] = 'o
     for _ in range(counter):
         for f in funcs:
             test_name = f.__name__
-            try:
-                os.makedirs(os.path.join(output, SUITE_NAME, test_name))
-            except FileExistsError:
-                pass
+            if SCREENSHOTS_ON:
+                try:
+                    os.makedirs(os.path.join(output, SUITE_NAME, test_name))
```

Comment:
prefer using pathlib.Path instead os.path and os.makedirs(use  Path.mkdir(mode=0o777, parents=False, exist_ok=False))
```
p = output / SUITE_NAME / test_name
p.mkdir()

Reply:
fixed